scripts/spack/packages/axom/package.py

```diff
@@ -43,7 +43,7 @@ class Axom(CachedCMakePackage, CudaPackage, ROCmPackage):
     # Note: Make sure this sha coincides with the git submodule
     # Note: We add a number to the end of the real version number to indicate that we have
     #  moved forward past the release. Increment the last number when updating the commit sha.
-    version('0.6.1.5', commit='d38862ac909fccbfb3d828a920063b4be4ec6a70', submodules="True")
```

Comment:
Shouldn't the MFEM SHA also be changed?

Reply:
yes good catch.... too many things all at once